ccm_web/client/src/components/Home.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Grid from '@material-ui/core/Grid'
+
+import FeatureCard, { FeatureCardProps } from './FeatureCard'
+
+import AccountCircleOutlinedIcon from '@material-ui/icons/AccountCircleOutlined'
+import LibraryBooksOutlinedIcon from '@material-ui/icons/LibraryBooksOutlined'
+import MergeTypeIcon from '@material-ui/icons/MergeType'
+import PersonAddIcon from '@material-ui/icons/PersonAdd'
+import PersonAddOutlinedIcon from '@material-ui/icons/PersonAddOutlined'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    minWidth: 275
+  }
+}))
+
+function Home (): JSX.Element {
+  const classes = useStyles()
+
+  const mergeSectionProps: FeatureCardProps = {
+    title: 'Merge Sections',
+    description: 'Combine sections taught by yourself or others into one Canvas site for easier management',
+    icon: <MergeTypeIcon fontSize='large'/>,
+    size: 'tall'
+  }
+
+  const gradebookToolsProps: FeatureCardProps = {
+    title: 'Gradebook Tools',
+    description: 'Trim the gradebook from Canvas, or trim the gradebook from a third party to correct format',
+    icon: <LibraryBooksOutlinedIcon fontSize='large'/>,
+    size: 'short'
+  }
+
+  const addSectionsGroupsProps: FeatureCardProps = {
+    title: 'Add Sections / Groups',
+    description: 'Add sections, groups through csv viles into your own course',
+    icon: <AccountCircleOutlinedIcon fontSize='large'/>,
+    size: 'tall'
+  }
```

Comment:
I think we will reach a point where some of this data is coming from the backend. To prepare for that, maybe we want to just offload this into a JSON file, figure out how to map data we can expect to come from the backend with the front-end stuff (mostly the icons, I guess).

Reply:
The backend doesn't need to know about icon choices, but perhaps we can just use a simple object to map task (still thinking about the entity name) names or IDs to their icons.

Reply:
@ssciolla you mentioned some of these can come from the Backend, after LTI launch happens and based on the role are you going to send 
```
Person A, subaccount_admin, feature_list: all
Person B, instrutor, Feature-list: merge_sections, add_non_umich_user, Gradebook_tools
```

BE simple can send person role and UI can map it up to appropriate features. 

It can be done in either way, I am fine either way

Reply:
@pushyamig that was my thought of how it would work, with the UI determining what to display based on the roles it is passed.  That seems like a simple and logical and probably practical implementation to me.  

The text/icons coming from the backend seems like it could be a useful idea if that sort of thing was meant to be configurable or changing rapidly, or perhaps changing between releases.  I'm not sure  any of those things will be the case, though.

For now I just plan to leave this as is b/c I figure it's going to end up changing when we figure out how this is going to work.

Reply:
Yeah, that might be okay, @pushyamig, need to think about it a little more. There still might be advantages to keeping some of this data in a static file, rather than in a React component.